### DISCUSSION TITLE
What does this mean? Image Browser

### DISCUSSION THREAD

**Discussion by ronbere** [vladmandic/sdnext#1101] [Q&A] [answered]
Image Browser: ImageReward is not installed, cannot be used.

Windows10 , Chrome, last commit

**Comment by vladmandic** [comment]
its an optional plugin for image browser that is not installed by default and image browser exension prints that message.

**Reply by ronbere** [reply]
ok, i check for this extension or plugin...No luck

**Reply by Tillerz** [reply, marked as answer]
https://github.com/THUDM/ImageReward
